bin/prepare_project_edition.sh

```diff
@@ -36,15 +39,27 @@ echo "> Move ${DEPENDENCY_PACKAGE_DIR} to ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACK
 mkdir -p ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}
 mv ${DEPENDENCY_PACKAGE_DIR}/* ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/
 
+# Remove installed dependencies inside the package
+rm -rf ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/vendor
+
 # Go to main project dir
 cd ${PROJECT_BUILD_DIR}
 
+# Copy auth.json if needed
+if [ -f ./${DEPENDENCY_PACKAGE_NAME}/auth.json ]; then
+    cp ${DEPENDENCY_PACKAGE_NAME}/auth.json .
+fi
+
+if [[ "$PROJECT_EDITION" != "oss" ]]; then
+    composer config repositories.ibexa composer https://updates.ibexa.co
+fi
+
 echo "> Make composer use tested dependency"
 JSON_STRING=$( jq -n \
                   --arg packageVersion "$DEPENDENCY_PACKAGE_VERSION" \
                   --arg packageName "$DEPENDENCY_PACKAGE_NAME" \
                   --arg packageDir "./$DEPENDENCY_PACKAGE_NAME" \
-                  '{"type": "path", "url": $packageDir, "options": { "versions": { ($packageName): $packageVersion}}}' )
+                  '{"type": "path", "url": $packageDir, "options": { "symlink": false , "versions": { ($packageName): $packageVersion}}}' )
```

Comment:
I had to disable symlinking - form-builder assets did not build because of issues with relative paths.